### DISCUSSION TITLE
Welcome to google-maps-scraper Discussions!

### DISCUSSION THREAD

**Discussion by Chetan11-dev** [omkarcloud/google-maps-scraper#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by zaGuru** [comment]
Sponsored! Thank you!

**Reply by zaGuru** [reply]
my email is rewter@gmail.com

**Reply by Chetan11-dev** [reply]
Delivered! Let the community know your review!

**Comment by Nakatomy** [comment]
Sponsored! Many thanks!

**Reply by Chetan11-dev** [reply]
Delivered! Let the community know your feedback!

**Comment by hichem-code** [comment]
Sponsored!

**Reply by Chetan11-dev** [reply]
We couldn't locate your email address. To receive the Pro Version, please choose perform one of the following actions:

1. Share your email address in the discussion.
2. Send an email to chetan@omkar.cloud using the following template:

```
Subject: Request Access to Pro Version
Content: I have sponsored the project as YOUR_GITHUB_USERNAME. Please send the Pro Version.
```

Thank you!

**Reply by hichem-code** [reply]
my email is : hichem.code.fr@gmail.com

**Reply by Chetan11-dev** [reply]
Delivered! Let the community know your feedback!

**Comment by 007-JB** [comment]
~Ja - 2 days ago this was free and only pro version cost a load now you've commercialised the whole thing for more than the pro version £35 my a
